Changelog.md

```diff
@@ -90,6 +90,7 @@ Bugfixes:
  * Type Checker: Disallow assignments to mappings within tuple assignments as well.
  * Type Checker: Allow assignments to local variables of mapping types.
  * Type Checker: Consider fixed size arrays when checking for recursive structs.
+ * Type Checker: Fix crashes in erroneous tuple assignments in which the type of the right hand side cannot be determined.
```

Comment:
If this bug did not exist in the previous released version, I would not add a changelog entry.

Reply:
@axic confirmed #4711 in 0.4.24

Reply:
Actually the problem was somewhat disguised, since we use the empty tuple type as the void/error type :-). The rhs in #4711 will be the empty tuple and checkDoubleStorageAssignment will crash whether it's a wildcard assignment or not - and especially it also crashed in earlier versions.